packages/babel-helper-string-parser/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@babel/helper-string-parser",
+  "version": "7.18.6",
+  "description": "A utility package to parse strings",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/babel/babel.git",
+    "directory": "packages/babel-helper-string-parser"
+  },
+  "homepage": "https://babel.dev/docs/en/next/babel-helper-string-parser",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./lib/index.js",
+  "devDependencies": {
+    "charcodes": "^0.2.0"
+  },
```

Comment:
I don't know if this is bundled so asking to be on the safe side; should this be a normal dependency?

Reply:
We have a plugin that inlines `charcodes.***` into the corresponding codepoints: https://github.com/xtuc/charcodes/blob/master/packages/babel-plugin-transform-charcodes/README.md